frontend/views/containers/navigation/Profile.vue

```diff
@@ -2,18 +2,19 @@
 .c-profile(
   v-if='$store.state.loggedIn'
   data-test='userProfile'
-  )
-  .c-avatar-user
-    avatar-user(:username='ourUsername' size='sm')
-    .c-user
-      strong(
-        :data-test='userDisplayName ? "profileDisplayName" : "profileName"'
-      ) {{ userDisplayName ? userDisplayName : ourUsername }}
+)
+  profile-card(:username='ourUsername' :isSelf='true' direction='top-left')
+    .c-avatar-user
+      avatar-user(:username='ourUsername' size='sm')
+      .c-user
+        strong(
+          :data-test='userDisplayName ? "profileDisplayName" : "profileName"'
+        ) {{ userDisplayName ? userDisplayName : ourUsername }}
 
-      span(
-        data-test='profileName'
-        v-if='userDisplayName'
-      ) @{{ ourUsername }}
+        span(
+          data-test='profileName'
```

Comment:
This `<span />` needs to be a `<button />`, so that someone with a keyboard can interact with it and open the profile card.

The same should happen on the members' list. A way to detect these issues is by using the keyboard only and try to open and close the profile card. Use `Tab` to go forward and `Shift + Tab` to go backward.